### PR TITLE
Adding groups to profile

### PR DESCRIPTION
**Pull request by Brady76**
You can see what groups you're subscribed too and unsubscribe, however the items aren't disappearing after unsubscribing (unless you refresh) and they're not really aligned too well so that's next.

### CODE REVIEW

File: static/javascripts/components/profile.jsx [hunk 1]
```diff
@@ -7,17 +7,14 @@ getInitialState: function(){
     
     
     if(user_data[0].departments.length == 0 || user_data[0].departments == ['']){
-        console.log('default');
```
Comment (sungbae) [inline static/javascripts/components/profile.jsx]: No need to add these changes in, I'm still working in this file.

Comment (Brady76) [inline static/javascripts/components/profile.jsx]: I'll remove them later tonight

**Comment by Brady76** [comment]
@sungbae it's ready now

File: templates/profile.html [hunk 2]
```diff
@@ -4,12 +4,18 @@
   var user_data={% raw user_info_json %};
   var routesObject = {
       surveys: "/api/surveys",
-      response: "/api/response"
+      response: "/api/response",
```
Comment (sungbae) [inline templates/profile.html:7]: Do we need the surveys/response here? I thought this page will just include groups/subscribe.

File: static/javascripts/components/profile_groups.jsx [hunk 3]
```diff
@@ -0,0 +1,174 @@
+var ProfileGroups = React.createClass({
+    getInitialState: function(){
+        return({
+            currentGroups: [],
```
Comment (sungbae) [inline static/javascripts/components/profile_groups.jsx:4]: this should be `currentGroups: user_data[0].subscribed_groups` no? Cuz how else are you going to get groups subscribed from the database.

File: static/javascripts/components/profile_groups.jsx [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+                        currentGroups: data,
+                    })
+                }.bind(this),
+                error: function(xhr, status,err){
+                    console.log('error');
+                }.bind(this)
+        });
+    },
+    
+    componentDidMount: function(){
```
Comment (Brady76) [inline static/javascripts/components/profile_groups.jsx:48]: This function right here gets your subscribed groups @sungbae

File: templates/profile.html (hunk 2 above)
Comment (Brady76) [inline templates/profile.html:7]: No, but it's not harmful.

**Pull request merged**